corehq/apps/reports/scheduled.py

```diff
@@ -7,7 +7,10 @@
 from six.moves import range
 
 
-_soft_assert = soft_assert(exponential_backoff=True)
+_soft_assert = soft_assert(
+    to='{}@{}'.format('supportteam', 'dimagi.com'),
```

Comment:
The subject of this PR says "devs", but this says "supportteam". Was that intentional?

Reply:
yeah, it was originally just going to go to commcarehq ops, but then I added supportteam at Kai's request as well.  commcarehq ops should still get cc'd.